caa2021_packages_abstract.md

```diff
@@ -1,33 +1,38 @@
 ---
-title: "Tools for the revolution: developing scientific programming packages for archaeology"
+title: "Tools for the 'Third Science Revolution': developing scientific software packages for archaeology"
 author:
 - Joe Roe <<joe@joeroe.io>>, University of Copenhagen  
 - Martin Hinz <<martin.hinz@iaw.unibe.ch>>, University of Bern  
-- Clemens Schmid <<clemens@nevrome.de>>, MPI-SHH
+- Clemens Schmid <<schmid@shh.mpg.de>>, MPI-SHH
 output: pdf_document
 bibliography: references.bib
+geometry: "left=1.5cm,right=1.5cm,top=1cm,bottom=1cm"
 ---
 
 Standard session proposal for CAA2021, Cyprus  
 Convened on behalf of the *Scientific Scripting Languages in Archaeology* special interest group (SIG-SSLA)
 
 ## Abstract
 
-The increasing use of scientific programming languages (e.g. R or Python) is transforming the practice of computational archaeology.
-This "tool-driven revolution" [@Schmidt2020] promises to greatly improve the accessibility, power, and reproducibility of quantitative analyses.
-But a tool-driven revolution needs tools.
+The increasing use of scientific programming languages (e.g. R or Python) is changing the way computational analysis are conducted in archaeology.
+This "tool-driven revolution" [@Schmidt2020] promises to greatly improve the accessibility, power, and reproducibility of quantitative analyses and thereby to permanently influence the field as a whole [@Kristiansen2014].
+
+But a tool-driven revolution dies without versatile and sturdy tools.
 As inveterate methodological borrowers, we can frequently rely on implementations in other fields, but the adoption of scripted analysis also reiterates the long-established need for methods designed specifically for archaeological data and archaeological problems [@Kintigh1987;@Aldenderfer1998].
-As a result, recent years have seen a proliferation in packages developed by and for archaeologists (<<http://open-archaeo.info/>>).
+Recent years have seen a proliferation in packages developed by and for archaeologists (e.g. <<http://open-archaeo.info/>>).
```

Comment:
Open Archaeo is not working any longer!?

Reply:
I haven't updated open-archaeo in a while due to it being not as big of a priority for me throughout the summer and autumn of 2020. But this will give me reason to finally get things going again, including a new tagging feature (rather than categories or headings) that I've been planning for a while.

Reply:
It was working on Friday! Let's assume Zack will have it fixed before the conference :)